jams/eval.py

```diff
@@ -54,10 +54,7 @@ def validate_annotation(ann, namespace):
         If `ann` fails schema validation
     '''
 
-    if not ann.search(namespace=namespace):
-        raise NamespaceError('Expected namespace="{:s}", found "{:s}"'
-                             .format(namespace, ann.namespace))
-
+    ann = convert(ann, namespace)
```

Comment:
> So I then try to use this annotation with the transcription eval wrapper, which expects pitch_hz annotations, and it breaks.

Ah, I see what you're saying.  I forgot to change the return type.

Perhaps this should be renamed to `coerce_annotation`.